source/patterns/@aws-solutions-constructs/aws-wafwebacl-cloudfront/test/test.wafwebacl-cloudfront.test.ts

```diff
@@ -0,0 +1,317 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+// Imports
+import * as cdk from "@aws-cdk/core";
+import { WafwebaclToCloudFront } from "../lib";
+import * as cloudfront from "@aws-cdk/aws-cloudfront";
+import * as origins from '@aws-cdk/aws-cloudfront-origins';
+import * as s3 from "@aws-cdk/aws-s3";
+import * as waf from "@aws-cdk/aws-wafv2";
+import * as defaults from '@aws-solutions-constructs/core';
+import '@aws-cdk/assert/jest';
+
+function deployConstruct(stack: cdk.Stack, constructProps?: waf.CfnWebACLProps) {
+  const myBucket = new s3.Bucket(stack, 'myBucket');
```

Comment:
Send props that specify a REMOVAL-POLICY: DESTROY on this, or it will leave behind S3 buckets that will need to be cleaned up manually (one of the issues keeping us out of Nightswatch)